cmd/operator-verify/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	manifests "github.com/operator-framework/api/cmd/operator-verify/manifests"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "operator-verify",
+		Short: "Operator manifest validation tool",
+		Long:  `operator-verify is a CLI tool that calls functions in pkg/validation.`,
+	}
+
```

Comment:
what do you think of adding an option to output the resulting errors/warnings to a log file - I think this was a use case that Ralph or Scott had with operator courier

Reply:
I also think this can be a follow-up.